src/models/enums/MessageCodes.ts

```diff
@@ -7,7 +7,7 @@ export enum MessageCodes {
 	UNRECOGNISED_CLIENT = "UNRECOGNISED_CLIENT",
 	FAILED_DECRYPTING_JWE = "FAILED_DECRYPTING_JWE",
 	FAILED_DECODING_JWT = "FAILED_DECODING_JWT",
-	FAILED_VERIFYING_JWT = "FAILED_VERIFYING_JWT",
+	BAV_FAILED_VERIFYING_JWT = "BAV_FAILED_VERIFYING_JWT",
```

Comment:
We shouldnt need to be distinct here we should know it was BAV from the account number etc in the logs